src/Lifecycle.js

```diff
@@ -389,7 +389,7 @@ function _persistCredentialsToLocalStorage(credentials) {
  * Logs the current session out and transitions to the logged-out state
  */
 export function logout() {
-    if (MatrixClientPeg.get().isGuest()) {
+    if (MatrixClientPeg.get() && MatrixClientPeg.get().isGuest()) {
```

Comment:
errrp this is just going to throw at L405 instead

Reply:
oops, so it will